test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -20,7 +20,6 @@
 
```

Comment:
There is a type (capabability) in the subject line of the commit message. How about:
validation: tm: check egress capababilities before running tests

Reply:
I managed to copy paste the original typo in my comment (sorry about that) and now it got copy pasted in the latest comment, so the subject line still contains "capababilities".

Reply:
Sorry, I also missed to read it while copy pasting.